### PR TITLE
Scaffold for CLI

### PR DESCRIPTION
**Pull request by pkelv**
#55

### CODE REVIEW

File: cli/cmd/create.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var storage string
+var backupName string
+var clusterName string
```
Comment (earlgreyz) [inline cli/cmd/create.go]: This could be merged into one `var` block

File: cli/cmd/create.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+var createClusterCmd = &cobra.Command{
+	Use:   "cluster",
+	Short: "short description of create cluster",
+	Long: `A longer description of create cluster with examples:
+msp create cluster "my-cluster" --storage 1Gi`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			fmt.Println("Name of cluster is required")
```
Comment (earlgreyz) [inline cli/cmd/create.go]: `"Cluster name is required"`

File: cli/cmd/create.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+			return
+		}
+
+		if storage == "" {
+			fmt.Println("--storage is required")
+			return
+		}
+
+		if storage != "" && backupName != "" {
```
Comment (earlgreyz) [inline cli/cmd/create.go]: `storage != ""` check seems pointless here

File: cli/cmd/create.go [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+var createBackupCmd = &cobra.Command{
+	Use:   "backup",
+	Short: "short description of create backup",
+	Long: `A longer description of create backup with examples:
+msp create backup "my-backup" --cluster "my-cluster"`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			fmt.Println("Backup name is required")
```
Comment (aswietlik) [inline cli/cmd/create.go]: Backup name should be optional. We will use default name if not provided.

File: cli/cmd/update.go [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+// updateCmd represents the update command
+var updateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "A short description of update",
+	Long: `A longer description of update with usage:
+msp udpate cluster`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			fmt.Println(`You must specify a type of resource:
+- cluster`)
```
Comment (aswietlik) [inline cli/cmd/update.go]: We can update both cluster and backup.

File: cli/cmd/delete.go [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+// deleteCmd represents the delete command
+var deleteCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "A short description of delete",
+	Long: `A longer description of delete with examples:
+msp delete cluster`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			fmt.Println(`You must specify a type of resource:
+- cluster`)
```
Comment (aswietlik) [inline cli/cmd/delete.go]: Deleting a backup should also be possible.

File: cli/cmd/create.go [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+var createClusterCmd = &cobra.Command{
+	Use:   "cluster",
+	Short: "short description of create cluster",
+	Long: `A longer description of create cluster with examples:
+msp create cluster "my-cluster" --storage 1Gi`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			fmt.Println("Cluster name is required")
```
Comment (earlgreyz) [inline cli/cmd/create.go]: `fmt.Fatal` may be used instead

File: cli/cmd/root.go [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "msp",
+	Short: "MySQL Operator",
+	Long: `MySQL Operator is a command line interface
+that allow you to manage and create your cluster`,
```
Comment (earlgreyz) [inline cli/cmd/root.go]:
```diff
-allow
+allows
-your cluster
-MySQL Clusters and Backups
```

File: cli/cmd/root.go [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+	Short: "MySQL Operator",
+	Long: `MySQL Operator is a command line interface that allows
+you to manage and create MySQL Clusters and Backups`,
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
```
Comment (earlgreyz) [inline cli/cmd/root.go]: Maybe print to `stderr`?

**Pull request merged**